First draft of full Greenbone CE for Kali Linux documentation (#437)
greenbone/docs: src/22.4/kali/file-system.md

## Log And Configuration File Locations

> Here are the locations of the various Greenbone CE log and configuration files.

Reviewer: ```suggestion
Here are the locations of the various Greenbone Community Edition log and configuration files.
```

```{code-block}

> :caption: View all Greenbone CE log files

Reviewer: ```suggestion
:caption: View all Greenbone Community Edition log files
```

ls /var/log/gvm
```

```{code-block}

> :caption: View the OpenVAS configuration files

Reviewer: ```suggestion
:caption: View the OpenVAS Scanner configuration files
```

ls /etc/openvas
```

```{code-block}

> :caption: View the other Greenbone CE configuration files

Reviewer: ```suggestion
:caption: View the other Greenbone Community Edition configuration files
```
